### PR TITLE
Backend migration setup

### PR DESCRIPTION
**Pull request by Metemi**
Marked fixed it, it is working.

### CODE REVIEW

File: Backend-API/src/main/resources/db/migration/V1__create_table_users.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS users
+(
+    id   SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
```
Comment (Normelik) [inline Backend-API/src/main/resources/db/migration/V1__create_table_users.sql:4]: shoould be email and password?

Comment (Metemi) [reply Backend-API/src/main/resources/db/migration/V1__create_table_users.sql:4]: Well could be, but now it is, in my opinion, just to set the migration. Than we can create more scripts. And create correct tables with correct values. After this merge, We can create new branch and base on values in User class update table or create new one

**Comment by Metemi** [comment]
After Mark fixed branch, it past CI

File: Backend-API/src/main/resources/db/migration/V1__create_table_users.sql (hunk 1 above)
Comment (markkovari) [reply Backend-API/src/main/resources/db/migration/V1__create_table_users.sql:4]: You can use `SELECT 1 + 1;` if you want to have less invasive query or changes and still want to run something.

**Review by markkovari — APPROVED** [review]

**Review by Normelik — APPROVED** [review]

**Pull request merged**